setup.py

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of Viper - https://github.com/viper-framework/viper
+# See the file 'LICENSE' for copying permission.
+from __future__ import unicode_literals
+from __future__ import absolute_import
+from __future__ import print_function
```

Comment:
Same here, are you sure it is useful? I prefer fixing the code to a python >=2.7 syntax than adding imports for somewhat broken code.